machine_configuration/README.md

```diff
@@ -87,6 +87,18 @@ $ lsmod | grep ast
 ```
 Ensure that ast driver is not loaded.
 
+# Machine Configuration for Provisioning Intel QAT
+
+* Turn on `intel_iommu` kernel parameter for QAT and load `vfio-pci` at boot
+
+For QAT, We need to turn on `intel_iommu` kernel parameter. Also we need to load `vfio-pci` before we can deploy the QAT plugin. Both can be done my same MCO using the following command. 
+
+```
+$ oc apply -f https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/100-intel-qat-intel-iommu-on.yaml
+```
+
+Note: This will reboot the QAT worker nodes one by one to turn on intel_iommu kernel parameter.
```

Comment:
Can we change it to
Note: This will reboot the QAT worker nodes when changing the kernel parameter through MCO.

Reply:
Done